server/mergin/sync/utils.py

```diff
@@ -404,3 +404,30 @@ def get_order_param(cls: Model, order_param: OrderParam) -> Optional[UnaryExpres
 def is_valid_gpkg(file_meta):
     """Check if diff file is valid"""
     return file_meta["size"] != 0
+
+
+def parse_order_params(cls, order_params):
```

Comment:
I think this is generic function for any model and it would be better if it is somewhere in top directory not in sync module.